src/templates/cookies.njk

```diff
@@ -0,0 +1,105 @@
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% extends 'layout.njk' %}
+
+{% set pageTitle = 'Cookies' %}
+
+{% block content %}
+
+  {% if cookiePageConfirmation %}
+    {% set cookieNotificationBannerHtml %}
+      <h3 class="govuk-notification-banner__heading">
+        You’ve set your cookie preferences.
+      </h3>
+    {% endset %}
+
+    {{ govukNotificationBanner({
+      html: cookieNotificationBannerHtml,
+      type: "success"
+    }) }}
+  {% endif %}
+
+  <form action="/cookies" method="post">
+    <h1 class="govuk-heading-l">Cookies</h1>
```

Comment:
Could we move all the non form related stuff outside of the form itself? Can potentially cause issues with assistive technologies.